### PR TITLE
Rely on the Blob and ObjectURL API.

### PR DESCRIPTION
**Pull request by rgbkrk**
This makes use of the [Blob API](https://developer.mozilla.org/en-US/docs/Web/API/Blob) and [`URL.createObjectURL`](https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL) to create well formed, fully parsed and executed blocks of HTML.

### CODE REVIEW

**Comment by rgbkrk** [comment]
Eh, this works about as well as `iframe`s, in that `embed` is also isolated from a script context.

**Comment by rgbkrk** [comment]
As for test failing, that's a result of JSDom not having `URL` defined.

**Comment by rgbkrk** [comment]
Ok, *now* I want to jump off of JSDom because there isn't support for this and [`createObjectURL` is in all the browsers](http://caniuse.com/#search=createObjectURL).

**Comment by rgbkrk** [comment]
Sweet. Found a way to create a proper document fragment with full context that the notebook expects:

```
> window.x = "It's ALIIIVE";
> var range = document.createRange();
> var fragment = range.createContextualFragment('<script>console.log(window.x)</script>'));
> document.body.appendChild(fragment);
It's ALIIIVE
```

File: src/html.transformer.js [hunk 1]
```diff
@@ -1,9 +1,7 @@
 'use strict'
 
 export function HTMLTransformer (mimetype, data, document) {
-  var el = document.createElement('div')
-  // TODO: Pull scripts from inside, create elements for them
-  el.innerHTML = data
-  return el
+  var range = document.createRange()
+  return range.createContextualFragment(data)
```
Comment (jdfreder) [inline src/html.transformer.js]: heh!  What a funny looking function name.

**Comment by jdfreder** [comment]
> Sweet. Found a way to create a proper document fragment with full context that the notebook expects:

JSDom supports this?

**Comment by rgbkrk** [comment]
No JSDom support for this either. :unamused: 

Time to switch over to Karma. Would you be able to do that Jon?

**Comment by jdfreder** [comment]
Yup, haven't played with Karma yet, but I'd like to get my hands wet with it tonight!

**Comment by karissa** [comment]
Might I nudge using tape+zuul? Expert node.js devs use it instead of karma https://github.com/nteract/transformime/pull/32/files#diff-b1f5e5df2ba50ea9660b908498761761R1

**Comment by jdfreder** [comment]
I haven't used either, so I'm unqualified to comment!  @rgbkrk , thoughts?  I'm fine using @karissa 's implementation.

So if we merge this, are we committing to not-supporting JSDom?  Are we sure we want to do that?

**Comment by rgbkrk** [comment]
I'm fine with it if you submit a patch that makes test pass, @karissa. That's all that is waiting on #32 from my perspective.

**Comment by rgbkrk** [comment]
Deferring to #32.

**Pull request closed without merging**